Knittery/Network/Codables/Photo.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Photo.swift
+//  Knittery
+//
+//  Created by Nick on 2022-10-11.
+//  Copyright © 2022 Nicholas Valenti. All rights reserved.
+//
+
+import Foundation
+
+struct Photo: Codable {
+    let id: Int?
+    let sortOrder: Int?
+    let thumbnail: String?
+    let small: String?
```

Comment:
I think this would be better named `smallURL `for clarity

Reply:
Done with a todo to maybe make it a URL? type instead